ktxspec.adoc

```diff
@@ -1071,15 +1073,43 @@ have shaders that understand this channel layout.
 The bitstream of the UASTC data is described in Chapter 25 _UASTC
 Compressed Texture Image Format_ of <<KDF13>>.
 
+Basis Universal ASTC HDR Format::
+The Universal ASTC HDR image format (UASTC HDR) is indicated by
+`colorModel` `KHR_DF_MODEL_UASTC_HDR` (= 167). This format supports
+two compression ratios defined by the texel block size in the DFD:
+4x4 and 6x6. Images in this format are a strict RGB-only ASTC HDR
+subset so `vkFormat` must be set to whichever of
+`VK_FORMAT_ASTC_4x4_SFLOAT_BLOCK` (= 1000066000) or
+`VK_FORMAT_ASTC_6x6_SFLOAT_BLOCK` (= 1000066004) matches the texel
```

Comment:
Is raw ASTC 6x6 still expected given the "GPU Photo" release?

Reply:
As I understand it, yes. @richgel999 please confirm.

Reply:
> Is raw ASTC 6x6 still expected given the "GPU Photo" release?

Yes, we're supporting raw ASTC 6x6 HDR with or without RDO (and with or without Zstd supercompression), in addition to the custom intermediate format mode (which nearly always gets higher compression). Internally we create the raw ASTC data anyway in the new compressor so this is easy for us.

Reply:
This section refers to UASTC HDR which originally supported transcoding to BC6H only for 4x4 blocks. Are you saying that ASTC HDR 6x6 transcoding should also be expected?

Reply:
> This section refers to UASTC HDR which originally supported transcoding to BC6H only for 4x4 blocks. Are you saying that ASTC HDR 6x6 transcoding should also be expected?

What we used to call "UASTC HDR" is now called "UASTC HDR 4x4" (which can be rapidly transcoded to BC6H). 

The latest release supports transcoding both plain ASTC HDR 6x6 (with or without RDO encoding), or ASTC HDR 6x6 intermediate (the custom format), to BC6H.

Reply:
I prefer to document these in one _\<somename HDR\>_ section, with the block size being the distinguishing feature, as I have done here. @richgel999 is there any reason this cannot work?

Reply:
> I prefer to document these in one _<somename HDR>_ section, with the block size being the distinguishing feature, as I have done here. @richgel999 is there any reason this cannot work?

I think that's fine. Also, I've basisu github, I've changed most references to "ASTC HDR 6x6" to "UASTC HDR 6x6" or "UASTC HDR 6x6 Intermediate", to avoid trademark issues. I'm working on updating our library enums.

Our "UASTC HDR 6x6" is constrained ASTC HDR 6x6, but still 100% standard ASTC HDR.
UASTC HDR 6x6 Intermediate ("GPU Photo") is a custom format.